sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Discount.java

```diff
@@ -25,10 +26,11 @@
     )
     private Set<Tag> tags = new HashSet<>();
 
-    @ManyToMany(mappedBy = "discounts")
-    private List<Employee> employee = new ArrayList<>();
+//    @ManyToMany(mappedBy = "discounts")
+//    private List<Employee> employee = new ArrayList<>();
```

Comment:
Let's remove this commented out code. Anyway we will use collection of EmployeeDiscount obejcts if it will be needed

Reply:
Ok. It was in code. I didn't touch it.